js/capture.js

```diff
@@ -44,7 +46,23 @@ var Logger = function() {
             }
             break;
         }
+        if (r) {
+            this.DOMAIN_RESTRICTION = r;
+        }
     }
+
+    this.oneEquals = function(optionsStr, target) {
```

Comment:
This can be greatly simplified to

```js
this.oneEquals = function(optionsStr, target) {
  if (!optionsStr || !target) return false;
  return optionsStr.split(' ').indexOf(target) !== -1;
}
```